docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.7"
+
+#    This Source Code Form is subject to the terms of the Mozilla Public License,
+#    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+#    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+#    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#    
+#    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+#    graphic logo is a trademark of OpenMRS Inc.
+
+services:
+    db:
+        image: mariadb:10.3
```

Comment:
@rkorytkowski  @dkayiwa  just asking for learning's sake .
Are we not including **postgres** , How do we include **postgres** as an alternative for those wishing not wishing to run **mariadb** / mysql /

Reply:
I think it's safe to target mariadb over msyql. I'd actually like to migrate to postgresql as our first choice, but I think the majority of implementing teams have MySQL experience and that translates more easily (hopefully seamlessly) to mariadb.

Reply:
My guess if you were to poll the community is that 95% of implementations are using MySQL 5.6 or 5.7.  MariaDB might be the right choice here, but I would be surprised to learn if it is widely used over MySQL.  If we want people to have the _most_ familiar environment to start out with, and the most well-tested / supported setup, it is definitely MySQL.

Reply:
@mseaton the last time I checked there was no mysql build for ARM architecture (Apple's M1) so it's not ideal for devs (requires emulation). It most likely has performance implications, but if devs are not testing against a huge dataset (which is most of the time) then we should be fine. I haven't assessed differences between the two in terms of replication / sharding / partitioning. This is where I envisioned for us to provide configuration and it would be good to have it for the most used DB or the DB that we want most of implementations to use.

Reply:
@rkorytkowski - why do devs need a MySQL build?  Isn't running MySQL in Docker the way to go?  In any event, I'm not necessarily advocating here, just saying that if we are trying to reflect what is _actually_ used and best supported, then my guess is that it's still MySQL by far.

Reply:
By MySQL build I mean a MySQL image built to be run on the given CPU, which can be used on devs' machines. When using Docker you still need to get the right image for your CPU architecture x86, AMR, etc. to run it or else it needs to be emulated.